OxfordBodleian/Juel-Jensen/BDLaethe37.xml

```diff
@@ -158,100 +1601,230 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <incipit xml:lang="gez">
+                                    <locus target="#95ra"/>
+                                    ዘስላሤ፡ ስብሐት፡ ለክሙ፡ ስሉስ፡ ቅዱስ፡ ማዕምራነ፡ ክልኤ፡ አእምሮ፡ ለብሃን፡
+                                    ዓቢይ፡ ዘፈጠሮ፡ ቃልክሙ፡ በተናግሮ። ስብ፡ ሰዓተ፡ ሌሊት፡ ስላሤ፡ ዘወሰንክሙ፡
+                                    በጎህ፡ ወዘአ<unclear>ክ</unclear>በርክሙ፡ በጸዳል፡ ብሩህ፡ ፀሐየ፡ እምነ፡ ወርኅ።
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <locus from="95ra" to="95rv"/>
+                                    ስ፡ ከመ፡ ኀብረ፡ ሰደ፡ ሥጋየ፡ አመ፡ በኃጢአት፡<pb n="95r"/><cb n="b"/> <unclear>ቄ</unclear>ሐ፡ ኀዘን፡
+                                    ብየ፡ <add place="above">አምጣነ፡</add> በዝሐ፡ ሥላሴ፡ ዘዮም፡ ወዘአሜሃ፡ በጽባሕ፡ ሀቡኒ፡ 
+                                    ፍሥሐ<hi rend="rubric" rendition="#partialRubric">።።።።</hi> ስብሐት፡ በል፡
+                                </explicit>
+                            </msItem>
 
-                <!--    <history>
-                        <origin>
-                            <origDate when="" evidence=""/>
-                        </origin>
-                        <provenance>  </provenance>
-                    </history>-->
-
-                    <additional>
-                        <adminInfo>
-                            <recordHist>
-                                <source>
-                                  <!--  <listBibl type="secondary">
-                                        <bibl></bibl>
-                                    </listBibl>-->
-                                </source>
-                            </recordHist>
-                        </adminInfo>
-                    </additional>
-
+                        </msItem>
+                        </msContents>
+                        
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                        <p>A hole from the parchment making process has been repaired by sewing, e.g.
+                                            <locus target="#95"/>.</p>
+                                    </support>
+                                    <foliation>
+                                    </foliation>
+                                    <collation>
+                                        <list>
+                                        <item xml:id="q11">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="94r" to="95v"/>
+                                        </item>
+                                        </list>
+                                    </collation>
+                                    <condition key="good"></condition>
```

Comment:
Do we want to repeat the condition element in each text part? I would rather keep it in a single element since most of the time, damage occurred to the manuscript as a whole...but we can discuss it separately.

Reply:
I was also wondering about this. Maybe in some cases it can make sense to have it separate (if one part is more heavily damaged than another), but in this case I think it would make sense to only have it once for the entire object. …but yes, it's probably better to do it as part of the general overview of "condition"